.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @lokm01 @benedeki @DzMakatun @HuvarVer @dk1844 @AdrianOlosutean
+* @benedeki @dk1844 @AdrianOlosutean @Zejnilovic
```

Comment:
I think these people are the ones contributing the most to the project there days...

Reply:
```suggestion
* @benedeki @dk1844 @AdrianOlosutean
```

👍